projects/plugins/boost/tests/e2e/helpers/common-helper.js

```diff
@@ -0,0 +1,17 @@
+import { execWpCommand, resolveSiteUrl } from 'jetpack-e2e-commons/helpers/utils-helper.cjs';
+import { PostFrontendPage } from 'jetpack-e2e-commons/pages/index.js';
+
+export async function findPostIdByTitle( postTitle ) {
+	const cmd = `db query 'SELECT ID FROM wp_posts WHERE post_title LIKE "%${ postTitle }%" AND post_status="publish"' --skip-column-names`;
+	return await execWpCommand( cmd );
+}
+
+export async function visitPageByTitle( page, title ) {
```

Comment:
Yes, this was a leftover from a previous iteration. I have now removed the whole helper file in eb6a173 since I do not make use of the TestcontentPage anymore.